src/test/java/core/basesyntax/db/strategy/handler/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.db.strategy.handler.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.model.FruitTransaction;
+import core.basesyntax.db.strategy.OperationHandler;
+import core.basesyntax.db.strategy.OperationStrategy;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    private static OperationStrategy operationStrategy;
+
+    @BeforeClass
+    public static void setUp() {
+        operationStrategy = new OperationStrategyImpl();
+    }
+
+    @Test
+    public void get_supplyOperationTest_ok() {
```

Comment:
```suggestion
    public void get_supplyOperation_ok() {
```
 the same with other methods